### PR TITLE
Feature/mark node - PST-112

### PR DESCRIPTION
**Pull request by AlbinaMuzafarova**
- node marked once when clicked, twice once the extended preview seen. 
- marks saved to localstorage 

Note: didn't manage to change the background-color of the marked nodes. Any suggestions on a better approach are welcome!!

### CODE REVIEW

**Review by duaakelzi — APPROVED** [review]
works very well i like it thank you Abina

**Review by micahh2 — APPROVED** [review]
Awesome Albina!

File: client/src/components/ResultVisualization.vue [hunk 1]
```diff
@@ -521,7 +524,16 @@ export default defineComponent({
                 .selectAll('circle')
                 .classed('selected', false)
                 .filter((td) => td === node)
+                //highlight border
                 .classed('selected', true);
+
+            this.$emit('onPatentSelected', { patent: node.patent, index: node.index ?? -1 });
```
Comment (samuelhuebner) [inline client/src/components/ResultVisualization.vue:530]: This line causes the patent preview to be broken after clicking a non-patent node. Maybe it got here by accident during a merge?

**Review by samuelhuebner — CHANGES_REQUESTED** [review]
Works for me 👍🏼 Once you remove the line that causes the patent preview error we can merge that form my point of view

**Comment by AlbinaMuzafarova** [comment]
nice catch, thanks Samuel! Have removed the line and tested it on my machine - everything looks to work fine. Merging!

**Review by samuelhuebner — APPROVED** [review]

**Comment by samuelhuebner** [comment]
> nice catch, thanks Samuel! Have removed the line and tested it on my machine - everything looks to work fine. Merging!

🤩 🚀

**Pull request merged**